src/stores/ClientPresenceStore.js

```diff
@@ -21,7 +21,7 @@ class ClientPresenceStore extends PresenceStore {
   }
 
   async setClientPresence({ status, since, afk, activity }) { // eslint-disable-line complexity
-    const applicationID = activity && (activity.application ? activity.application.id || activity.application : null);
+    const applicationID = activity ? activity.applicationID || activity.application : null;
```

Comment:
just make it `(activity && activity.application) ? activity.application.id || activity.application : null;`, and make sure to update docs